bin/list-all-packages.sh

```diff
@@ -55,6 +55,7 @@ function list_other() {
     echo "docker-18.09.8.tar.gz bundles/"
     echo "docker-19.03.4.tar.gz bundles/"
     echo "docker-19.03.10.tar.gz bundles/"
+    echo "docker-19.03.15.tar.gz bundles/"
```

Comment:
new docker version?

Reply:
since docker 19.03.10 is unsupported on rhel 8 but the latest validated version for most kubernetes version is 19.03 i included a supported version

https://github.com/kubernetes/kubernetes/blob/v1.20.7/build/dependencies.yaml#L68-L70